codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/util/TypeConversionGenerator.kt

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package software.amazon.smithy.rust.codegen.util
```

Comment:
Code generator classes don't really belong in `util`. This should go in `software.amazon.smithy.rust.codegen.smithy.generators`.

Reply:
I will move it.